### PR TITLE
JC-2378 Fixed: new uploaded avatar i not shown

### PR DESCRIPTION
**Pull request by mihnayan**
Added version of user profile in URL of avatar
for showing new uploaded avatar.

### CODE REVIEW

File: jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/question.vm [hunk 1]
```diff
@@ -180,7 +180,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
         <div>
           <div class="question-left-panel pull-left">
             <div class="thumbnail wraptocenter">
-              <img src="${request.contextPath}/users/${post.userCreated.id}/avatar" alt=""/>
+              <img src="${request.contextPath}/users/${post.userCreated.id}/avatar?version=${post.userCreated.version}" alt=""/>
```
Comment (usharik) [inline jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/question.vm]: Does `alt=""` something useful?

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java [hunk 2]
```diff
@@ -248,6 +245,21 @@ public UserProfileDto getUserProfileDto() {
     }
 
     /**
+     * Set user profile version
+     * @param version - profile version
+     */
```
Comment (usharik) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: The name of method is quite informative, so this JavaDoc is redundant.

File: jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/editProfile.jsp [hunk 3]
```diff
@@ -122,7 +122,7 @@
             <img id="avatarPreview" src="data:image/jpeg;base64,${editedUser.avatar}" alt=""/>
           </c:when>
           <c:otherwise>
-            <img src="${pageContext.request.contextPath}/users/${editedUser.userId}/avatar" alt=""/>
+            <img src="${pageContext.request.contextPath}/users/${editedUser.userId}/avatar?version=${editedUser.userVersion}" alt=""/>
```
Comment (usharik) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/editProfile.jsp]: Does `alt=""` something useful?

File: jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp [hunk 4]
```diff
@@ -284,7 +284,7 @@
       </div>
              
                    <span class="thumbnail post-userinfo-avatal wraptocenter">
-                        <img src="${pageContext.request.contextPath}/users/${post.userCreated.id}/avatar" alt=""/>
```
Comment (usharik) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp]: Does `alt=""` something useful?

File: jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/question.vm (hunk 1 above)
Comment (mihnayan) [inline jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/question.vm]: I can't know. This part of code is not mine, it was being here before I changed the line. My changes only affect version of avatar.

File: jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/editProfile.jsp (hunk 3 above)
Comment (mihnayan) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/editProfile.jsp]: I can't know. This part of code is not mine, it was being here before I changed the line. My changes only affect version of avatar.

File: jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp (hunk 4 above)
Comment (mihnayan) [inline jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp]: I can't know. This part of code is not mine, it was being here before I changed the line. My changes only affect version of avatar.

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java (hunk 2 above)
Comment (mihnayan) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java]: Ok, I agree with you at this point.

**Pull request closed without merging**